python/psqlpy/_internal/__init__.pyi

```diff
@@ -321,6 +321,7 @@ class Transaction:
         self: Self,
         querystring: str,
         parameters: list[Any] | None = None,
+        prepared: bool = True,
```

Comment:
You sure, we want to use prepared statements by default?

Reply:
I think so cuz I assume that most people don't use PGBouncer which can break the prepared statement.
Plus, if someone uses PGBouncer only 2/3 pool strategies can break this behavior.

And at the same time prepared statements lead to a significant performance boost.